app/assets/javascripts/govuk.js

```diff
@@ -15,28 +15,24 @@ $(document).ready(function () {
   var showHide = new ShowHide()
   showHide.init()
 
-  if (document.getElementById('publisher')) {
-    accessibleAutocomplete.enhanceSelectElement({
-      selectElement: document.querySelector('#publisher'),
-      showAllValues: true,
-      preserveNullOptions: true
-    })
+  var autocompleteSelects = document.getElementsByClassName('js-accessible-autocomplete');
```

Comment:
From the look of the rest of this file, it looks as if it's been linted with [StandardJS](http://standardjs.com), which is the chosen linter from the [GDS Way rules for JavaScript](https://gds-way.cloudapps.digital/manuals/programming-languages/js.html#linting).

I think the new bits of JavaScript need to be linted as well to ensure that the file is consistent.

```suggestion
  var autocompleteSelects = document.getElementsByClassName('js-accessible-autocomplete')
```